src/core/operations/DeriveEVPKey.mjs

```diff
@@ -62,11 +62,13 @@ class DeriveEVPKey extends Operation {
      * @returns {string}
      */
     run(input, args) {
-        const passphrase = Utils.convertToByteString(args[0].string, args[0].option),
+        const passphrase = CryptoJS.enc.Latin1.parse(
```

Comment:
This feels a little contrived at the moment.  Using `Latin1` as the encoding scheme for the string does solve the issue, but doesn't feel optimal.  Could we instead replace this with a conversion to a byte array and then create a crypto word array from that?  E.g:

```javascript
// Note: Entirely untested.
CryptoJS.lib.WordArray.create(new UInt8Array(Utils.convertToByteArray(args[0].string, args[0].option)))
```

Reply:
That being said, this is a clear improvement over the current broken implementation, so if this isn't a two minute replacement I'm happy to approve and merge in this pull request.

Reply:
Doesn't WordArray expect an array of 32bit integers? I'm not sure whether `new Uint32Array(new Uint8Array(byteArray).buffer)` would have the correct endianness

Reply:
Looks like WordArray expects BigEndian version of the buffer.. The Latin1 `parse` code - at least to me - looks like something as direct as possible given we need an array of 32bit integers represending BE view of the string

Reply:
That's fair, I'll approve this then! 👍